saraheisa/Photons
# Issue#16 | Initialize DB with tables Schema

## Description

This PR creates tables Schema to initiate DB and creates a temporary endpoint to initiate DB on server

## Related Issue

closes #16 

## Acceptance Criteria

- [X] Create a file with a method to create all the table needed for the project
- [X] Set a command to initialize DB
- [X] Set a temporary endpoint to initialize DB on Amazon RDS

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |


## Requests / Responses

**Request**

GET `/initialize` Returns DB queries response

**Response**

HTTP/1.1 200 OK

OR

HTTP/1.1 500 Internal Server Error

## Testing Steps

- From your terminal, pull down this branch with git pull origin `Issue16-initialize-DB`
- Check that branch out with `git checkout Issue16-initialize-DB`
- Run `npm run dev` to run the app
- Go To [`http://localhost:3000/initialize`](http://localhost:3000/initialize)
